app/helpers/storage_resource_helper/textual_summary.rb

```diff
@@ -34,4 +35,12 @@ def textual_logical_free
   def textual_logical_total
     {:label => _("Total Size"), :value => number_to_human_size(@record.logical_total, :precision => 2)}
   end
+
+  def textual_capabilities
+    values = ""
+    JSON.parse(@record.capabilities).each do |capabilities|
+      values += "#{capabilities['abstract_capability']}: #{capabilities['value']}, "
```

Comment:
I'm not a fan of `abstract_capabilities` here, we should use something simple like `name` and if we have to convert it in the refresh parser we can that's exactly what it is for

Reply:
We can change it while parsing to `capability_name`.
is that ok ?

Reply:
`capability_name` in a column called `capabilities` is a little redundant.